src/main/java/org/broadinstitute/hellbender/engine/filters/ReadStrandFilter.java

```diff
@@ -6,7 +6,7 @@
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 
 /**
- * Keep only reads whose strand is forward or reverse
+ * Keep only reads whose strand is as specified.
```

Comment:
What do we mean by reverse or forward strand? Do we mean when viewing the reference in IGV, the sequence that is shown is the forward strand and it's reverse complement is the reverse strand?